src/Aspire.Dashboard/Components/Pages/Metrics.razor.cs

```diff
@@ -165,7 +169,10 @@ private Task HandleSelectedApplicationChangedAsync()
             }
         }
 
-        return this.AfterViewModelChangedAsync(_contentLayout, isChangeInToolbar: true);
+        // On mobile, we actually *do* want to update this immediately, since it will affect the tree of possible
```

Comment:
What's "this"? I'm guessing it's the toolbar, but the comment could be made clearer.

Reply:
changed comment